modules/gapi/test/cpu/gapi_stereo_tests_cpu.cpp

```diff
@@ -27,9 +27,9 @@ INSTANTIATE_TEST_CASE_P(CPU_Tests, TestGAPIStereo,
                                 Values(cv::gapi::StereoOutputFormat::DEPTH_FLOAT16,
                                        cv::gapi::StereoOutputFormat::DEPTH_FLOAT32,
                                        cv::gapi::StereoOutputFormat::DISPARITY_FIXED16_12_4,
-                                       cv::gapi::StereoOutputFormat::DEPTH_FLOAT16,
-                                       cv::gapi::StereoOutputFormat::DEPTH_FLOAT32,
-                                       cv::gapi::StereoOutputFormat::DISPARITY_Q16_11_4),
+                                       cv::gapi::StereoOutputFormat::DEPTH_16F,
+                                       cv::gapi::StereoOutputFormat::DEPTH_32F,
+                                       cv::gapi::StereoOutputFormat::DISPARITY_16Q_11_4),
```

Comment:
Do we really need to test both sets?

Reply:
yeah the "old" enums leaked already to the internal app we provide in packages.